### PR TITLE
docs: bring back missing descriptions

### PR DESCRIPTION
**Pull request by theo-mesnil**
Signed-off-by: Théo Mesnil <mesniltheo@gmail.com>

### CODE REVIEW

File: docs/components/Mdx/Showcase/Item.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 /* eslint-disable react/prop-types */
+import React from 'react'
 import { Box } from '@welcome-ui/box'
 import { CheckIcon } from '@welcome-ui/icons.check'
 import { CopyIcon } from '@welcome-ui/icons.copy'
 import { Text } from '@welcome-ui/text'
 import { useCopyText } from '@welcome-ui/utils.copy'
-import React from 'react'
+import { useTheme } from '@xstyled/styled-components'
```
Comment (cnairiwttj) [inline docs/components/Mdx/Showcase/Item.js:8]: I'm not sure you're using "useTheme" somewhere in this file

File: packages/Popover/index.tsx [hunk 2]
```diff
@@ -10,7 +10,12 @@ import { Trigger } from './Trigger'
 import { UsePopoverStateReturn } from './usePopoverState'
 
 export interface PopoverOptions {
+  /** call an function when popover closed */
```
Comment (cnairiwttj) [inline packages/Popover/index.tsx:13]: just a little typo : "call a function..."

File: docs/pages/components/emoji-picker.mdx [hunk 3]
```diff
@@ -229,7 +229,7 @@ function BasicList() {
 
 ### EmojiPicker
 
-<props propTypes={props.propTypes.EmojiPicker} />
+<props propTypes={props.propTypes.EmojiPicker} props={props.propTypes} />
```
Comment (mleralec) [inline docs/pages/components/emoji-picker.mdx:232]: le composant props n'utilise pas la prop "props" je crois 🤔

Comment (theo-mesnil) [reply docs/pages/components/emoji-picker.mdx]: yes un oubli de done :)

**Review by mleralec — APPROVED** [review]
:fire:

**Pull request merged**